packages/tupaia-web/src/features/Dashboard/Dashboard.tsx

```diff
@@ -189,6 +204,11 @@ export const Dashboard = () => {
       <Panel $isExpanded={isExpanded}>
         <ExpandButton setIsExpanded={toggleExpanded} isExpanded={isExpanded} />
         <ScrollBody>
+          <Snackbar anchorOrigin={{ vertical: 'top', horizontal: 'right' }} open={isSnackbarOpen} autoHideDuration={6000} onClose={handleCloseSnackbar}>
+            <Alert elevation={6} variant="filled" onClose={handleCloseSnackbar} severity="success">
+              {isSubscribed ? 'Succesfully joined dashboard mailing list' : 'You are now removed from this dashboard mailing list'}
+            </Alert>
+          </Snackbar>
```

Comment:
If it's okay, could we back out this stuff around the toast notifications for now?
We've got a separate ticket for them here: https://linear.app/bes/issue/RN-1063/toast-messages
And I'd probably recommend we want to take a different tack with them. Really happy to pair on that together :+1: